gallery/mysqlflexibleserver-insights/microsoft.dbformysql-flexibleservers.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
```

Comment:
this file's name, `mysqlflexibleserver-insights` implies that you are adding an "insights" view and not a "Workbooks" view?  if so, we have to make code changes/etc on the workbooks side to add this insights value as a valid value, hook up strings, etc. (if you're using it today, you should be seeing errors in the console that this is an unknown workbook type and it is being ignored.)

most insights don't generally show a gallery view, they are a single at scale or at resource template and have a link to the corresponding "workbooks" entry in their resource/etc.

are you testing this gallery with the instructions at https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Contributing.md#deploying-your-own-gallery ? or how are you testing it?

Reply:
@gardnerjr I'm testing this at resource level using empty template to make sure templates are working. And does naming effect the gallery items. Intent is to show all three templates as gallery templates when customer goes to workbooks from resource
![image](https://user-images.githubusercontent.com/50682039/132803590-2162acbe-1502-46f8-8d3c-26f28eaf3771.png)

Reply:
if you're intending to show these items in a workbooks view, the gallery file should be in the `workbook` folder, not in a custom insights folder.  if you do that, and use the instructions for hosting your own gallery, you should be able to see this live in your browser with the correct testing setup.

Reply:
@ericajlin i thought we had validation of some kind that the gallery files were using a known workbook type, is that not the case? (it doesn't seem to be?)

Reply:
We hosted the package in blob container and validated the gallery templates and working as expected.
https://ms.portal.azure.com/?feature.includePreviewTemplates=true&forceEnableFeatures=Microsoft.DBforMySQL%2FenableWorkbooks&feature.workbookGalleryRedirect=https%3A%2F%2Fprkintpackage.blob.core.windows.net%2Ftemplates%2Fpackage#home